File: Daily Challenge/Python/Evaluate Expressions - Words As Operators 13.1.2026.py
tokens=input().split()
res=int(tokens[0])

for i in range(1,len(tokens),2):
    op=tokens[i]
    val=int(tokens[i+1])
    if op=="plus": res+=val
    elif op=="minus": res-=val
    elif op=="multiply": res*=val
    elif op=="divide": res//=val

print(res)

File: Daily Challenge/Java/Evaluate Expressions - Words As Operators 13.1.2026.java
import java.util.*;
public class Hello {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        String[] input=sc.nextLine().trim().split(" ");
        int res=Integer.parseInt(input[0]);

        for(int i=1;i<input.length;i+=2){
            String op=input[i];
            int val=Integer.parseInt(input[i+1]);
            switch(op){
                case "plus": res+=val; break;
                case "minus": res-=val; break;
                case "multiply": res*=val; break;
                case "divide": res/=val; break;
            }
        }
        System.out.println(res);
    }
}